server.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: "WordPress Server: Install LEMP Stack with PHP 5.6 and MariaDB MySQL"
+  hosts: web
+  remote_user: root
```

Comment:
Not until `secure-root.yml` is either explicitly added to our instructions or merged into `server.yml` though.